Needs an 'autoconf', but after that it works.
diff -up openscap-0.6.0/configure.ac openscap-0.6.0/configure.ac
--- openscap-0.6.0/configure.ac	2010-07-14 15:35:31.100460874 -0400
+++ openscap-0.6.0/configure.ac	2010-07-14 15:35:38.024012796 -0400
@@ -278,8 +278,8 @@ CPPFLAGS="$CPPFLAGS_SAVE"
 
 AC_PATH_PROG(PERL, perl)
 PERL_INCLUDES="`$PERL -e 'use Config; print $Config{archlib}'`/CORE"
-vendorlib="$(  $PERL -e 'use Config; print $Config{vendorlib}'  | sed "s|$($PERL -e 'use Config; print $Config{prefix}')||" )"
-vendorarch="$( $PERL -e 'use Config; print $Config{vendorarch}' | sed "s|$($PERL -e 'use Config; print $Config{prefix}')||" )"
+vendorlib="$(  $PERL -e 'use Config; print $Config{vendorlib}'  | sed -e "s|$($PERL -e 'use Config; print $Config{prefix}')||" -e 's|/$||' )"
+vendorarch="$( $PERL -e 'use Config; print $Config{vendorarch}' | sed -e "s|$($PERL -e 'use Config; print $Config{prefix}')||" -e 's|/$||' )"
 AC_SUBST([PERL_INCLUDES], ["$PERL_INCLUDES"])
 AC_SUBST([perl_vendorlibdir], ['${prefix}'$vendorlib])
 AC_SUBST([perl_vendorarchdir], ['${prefix}'$vendorarch])
